### PR TITLE
Fix edit command timeslot

### PR DESCRIPTION
**Pull request by bentanjunrong**
Let's:
- Fix issue with parsing input when only start/end time of an exam/lesson are modified

Fixes:
- #142 
- #144 
- #153 
- #164

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/EditCommandParser.java [hunk 1]
```diff
@@ -110,7 +112,11 @@ public EditLessonCommand parseEditLesson(String args, GuiState guiState) throws
             editLessonDescriptor.setDay(ParserUtil.parseDay(argMultimap.getValue(PREFIX_DAY).get()));
         }
 
-        // TODO: Need to deal with case where only start or end time is given. Currently assume both given.
+        // Prevent users from inputting only either the start or the end timings
+        if ((argMultimap.getValue(PREFIX_START).isPresent() ^ argMultimap.getValue(PREFIX_END).isPresent())) {
```
Comment (rohit0718) [inline src/main/java/seedu/address/logic/parser/EditCommandParser.java:116]: i think no need for the double brackets here. either bracket like ((a)^(b)) or simply (a^b).

File: src/main/java/seedu/address/logic/parser/EditCommandParser.java [hunk 2]
```diff
@@ -156,7 +162,11 @@ public EditExamCommand parseEditExam(String args, GuiState guiState) throws Pars
             editExamDescriptor.setDate(ParserUtil.parseDate(argMultimap.getValue(PREFIX_DATE).get()));
         }
 
-        // TODO: Need to deal with case where only start or end time is given. Currently assume both given.
+        // Prevent users from inputting only either the start or the end timings
+        if ((argMultimap.getValue(PREFIX_START).isPresent() ^ argMultimap.getValue(PREFIX_END).isPresent())) {
```
Comment (rohit0718) [inline src/main/java/seedu/address/logic/parser/EditCommandParser.java:166]: same as above

**Review by rohit0718 — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/parser/EditCommandParser.java (hunk 1 above)
Comment (bentanjunrong) [reply src/main/java/seedu/address/logic/parser/EditCommandParser.java]: Yup my bad, removed the extra brackets

**Review by rohit0718 — APPROVED** [review]
lgtm! thank u sir

**Pull request merged**